public/css/mixin/progress-bar.less

```diff
@@ -177,3 +177,49 @@
     }
   }
 }
+
+.progress-bar.checks-disabled {
+  .above {
+    .checks-disabled-text {
+      display: block;
+      width: 100%;
+      text-align: center;
+    }
+  }
```

Comment:
This is no general style. Please move this to `check-statistics.less`.

Reply:
The disabled note is moved to overlay container now.